helm-projectile.el

```diff
@@ -64,7 +64,12 @@
 (defvar helm-projectile-current-project-root)
 
 (defcustom helm-projectile-truncate-lines nil
-  "Truncate lines in helm projectile commands when non--nil."
+  "Truncate lines in helm projectile commands when non--nil.
+
+Some helm-projectile commands have similar behavior with existing
+Helms. In these cases their respective custom var for truncation
```

Comment:
2 spaces after periods is [Emacs Lisp style](https://github.com/bbatsov/emacs-lisp-style-guide#comments).